Source/InstrumentEditorVRC7.cpp

```diff
@@ -430,7 +553,7 @@ void CInstrumentEditorVRC7::OnCopy()
 	int patch = m_pInstrument->GetPatch();
 	// Assemble a MML string
 	for (int i = 0; i < 8; ++i)
-		MML.AppendFormat(_T("$%02X "), (patch == 0) ? (unsigned char)(m_pInstrument->GetCustomReg(i)) : default_inst[patch * 16 + i]);
+		MML.AppendFormat(_T("$%02X "), (patch == 0) ? (unsigned char)(m_pInstrument->GetCustomReg(i)) : default_inst[m_iPatchTone][patch * 8 + i]);
```

Comment:
so you did the whole `patch * 8 + i` thing again. idk if it's practical to extract this logic into a separate method (referencing m_iPatchTone and taking parameters `patch`, and `byte`) or function (taking parameters `patch_bank_idx`, `patch`, and `byte`).

After deduplicating `CInstrumentEditorVRC7::LoadInternalPatch()`, this indexing logic appears 3 times total, so it may be worth extracting it to a function or method.